pkg/controllers/cloud/gcp/gke/reconcile.go

```
@@ -247,11 +276,7 @@ func (t *gkeCtrl) Reconcile(request reconcile.Request) (reconcile.Result, error)
 		}
 	}
 
-	if requeue {
-		return reconcile.Result{Requeue: true}, nil
-	}
-
-	return reconcile.Result{}, nil
+	return result, nil
```

we need to return the error, if there was any